### PR TITLE
PyCon UK 2017

### LINKED ISSUE #386: PyCon UK 2017
October 26-30, 2017

~~~ yaml
 - title: PyCon UK 2017
   dir:  pycon-uk-2017
   youtube_lists:
    # PYCON UK 2017: Thursday Room C
    - https://www.youtube.com/playlist?list=PLrkpavSsBQZ521U6MOejN-c795CY5JEaV
    # PYCON UK 2017: Thursday, Assembly Room
    - https://www.youtube.com/playlist?list=PLrkpavSsBQZ6wTGg-6qJQ_DZDynwKgcDu
    # PYCON UK 2017: Friday, Assembly Room
    - https://www.youtube.com/playlist?list=PLrkpavSsBQZ5FsR7YKGzJvTXDkD0OB4IZ
   language: eng
   related_urls:
   - label: schedule
     url: http://2017.pyconuk.org/schedule/
~~~

### PR DESCRIPTION
**Pull request by Daniel-at-github**
Fixes #386

### CODE REVIEW

File: pycon-uk-2017/videos/pycon-uk-2017-friday-opening-session.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "duration": 762,
+  "language": "eng",
+  "recorded": "2017-10-27",
+  "related_urls": [
+    {
+      "label": "event schedule",
+      "url": "http://2017.pyconuk.org/schedule/"
+    }
+  ],
+  "speakers": [],
```
Comment (redapple) [inline pycon-uk-2017/videos/pycon-uk-2017-friday-opening-session.json]: - Peter Inglesby (@inglesp I think)
- Carrie-Ann Philbin
- Daniele Procida
- Owen Campbell
- Nicholas Tollervey

File: pycon-uk-2017/videos/pycon-uk-2017-john-pinner-awards.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "duration": 1056,
+  "language": "eng",
+  "recorded": "2017-10-27",
+  "related_urls": [
+    {
+      "label": "event schedule",
+      "url": "http://2017.pyconuk.org/schedule/"
+    }
+  ],
+  "speakers": [],
```
Comment (redapple) [inline pycon-uk-2017/videos/pycon-uk-2017-john-pinner-awards.json]: - Nicholas Tollervey
- Peter Inglesby
- Sheila Pinner

Awarded:
- Vincent Knight
- Mary Mooney 
- Tim Golden
- Leona So
- Tony Ibbs
- Daniele Procida
- Éléonore Mayola
- Cat Lamin
- Josh Lowe

File: pycon-uk-2017/videos/pycon-uk-2017-jp-morgan-sponsor-session.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  "duration": 206,
+  "language": "eng",
+  "recorded": "2017-10-27",
+  "related_urls": [
+    {
+      "label": "event schedule",
+      "url": "http://2017.pyconuk.org/schedule/"
+    }
+  ],
+  "speakers": [],
```
Comment (redapple) [inline pycon-uk-2017/videos/pycon-uk-2017-jp-morgan-sponsor-session.json]: - Stephen Simmons

**Comment by Daniel-at-github** [comment]
@redapple, added all missing speakers

**Pull request merged**